jetbrains-core/src/software/aws/toolkits/jetbrains/core/explorer/AwsExplorerFactory.kt

```diff
@@ -47,6 +47,13 @@ class AwsExplorerFactory : ToolWindowFactory, DumbAware {
                             url = "https://github.com/aws/aws-toolkit-jetbrains"
                         )
                     )
+                    add(
+                        OpenBrowserAction(
+                            title = message("explorer.cut_issue"),
+                            icon = AllIcons.Vcs.Vendors.Github,
+                            url = "https://github.com/aws/aws-toolkit-jetbrains/issues/new/choose"
```

Comment:
We should pre-populate some of this based on things like their OS, toolkit version, etc: https://help.github.com/en/articles/about-automation-for-issues-and-pull-requests-with-query-parameters